benchexec/test_runexecutor.py

```diff
@@ -1199,6 +1199,125 @@ def test_uptime_without_lxcfs(self):
             uptime, 10, f"Uptime {uptime}s unexpectedly low in container"
         )
 
+    def test_fuse_overlay(self):
+        if not container.get_fuse_overlayfs_executable():
+            self.skipTest("fuse-overlayfs not available")
+
+        test_dir = "/tmp/fuse_test/"
```

Comment:
With something like this, one needs to be careful. Using hard-code paths, especially when in shared directories, is often problematic and should almost always be avoided.

One reason is security, i.e. guarding against [symlink attacks](https://en.wikipedia.org/wiki/Symlink_race). Another is robustness: This test is likely to fail if it is executed twice in parallel.

The correct way for temporary files and directories is to always use the specific functionality from the `tempfile` module for this. Here we should use `TemporaryDirectory` as in the other test.